src/components/models/Block1IntroTemplate.js

```diff
@@ -9,23 +9,23 @@ import { toFormattedText } from '../../services/formatting';
 const { muiGridBlockContainer } = stylesBase;
 const useStyles = makeStyles({
   muiGridBlockContainer,
-  container: {
+  container: ({ secondVersion }) => ({
     width: 'calc(100vw - 8px)',
     left: 'calc(-1 * (100vw - 100%) / 2)',
     position: 'relative',
-    background: colors.lightGrey,
-    border: '1px solid #DCDCDC',
+    background: secondVersion ? colors.white : colors.lightGrey,
```

Comment:
Let's give this a more descriptive name than `secondVersion`. I'm not sure what the best description is without seeing the two pages rendered side by side, but choose something that best describes the difference in styles? `borderVersion`. `whiteTextVersion`? Not sure what it is.

Reply:
I put `iconVersion` 
The first version images are photos and the second version image are icons